admin/listPurchases/listView.html

```diff
@@ -383,17 +399,25 @@
 }
 
 function checkForNewOrders() {
-    requestOrders((error, orders) => {
+    const requestParams = {
+        categories: ["completed"],
+        after: lastChecked,
+    };
+    requestOrders(requestParams, (error, data) => {
         if(error) {
             return console.error("Failed to poll for updates", error);
         }
-        if(orders.length > entries.length) {
-            let diffCount = orders.length - entries.length;
-            new Notification(`${diffCount} ${diffCount > 1 ? "nye ordrer" : "ny ordre"} til Mærkelex!`);
-            let newOrders = orders.filter(order => !entries.some(entry => entry.id == order.id));
+        lastChecked = new Date().toISOString();
+        const diffCount = + data.amount;
```

Comment:
`= +`? What does this mean?

This is actually equivalent to `const diffCount = data.amount` - the extra `+` does nothing in this context

Reply:
i see the confusion, due to bad formatting